himportscan/BUILD.bazel

```diff
@@ -12,10 +12,16 @@ haskell_toolchain_library(name = "base")
 
 haskell_toolchain_library(name = "bytestring")
 
+haskell_toolchain_library(name = "containers")
+
 haskell_toolchain_library(name = "directory")
 
 haskell_toolchain_library(name = "ghc")
 
+# Only necessary to be able to import GHC.Platform in HImportScan.GHC.Settings
+# TODO[GL]: we can probably remove this once we only support ghc >=9.2
+haskell_toolchain_library(name = "ghc-boot")
```

Comment:
Maybe this is redundant now.

Reply:
Unfortunately it isn't - it seems I introduced another dependency on this package without knowing. Updated the comment.